cmd/acb/validation_test.go

```diff
@@ -49,17 +49,17 @@ func TestValidateRegistryCreds_Invalid(t *testing.T) {
 }
 
 func TestValidatePush_Valid(t *testing.T) {
-	if err := validatePush(false, "", "bar", "qux"); err != nil {
+	if err := validatePush(false, []string{";bar;qux"}); err != nil {
```

Comment:
`bar` `foo` seems violating some global readiness policy. Maybe you can change to other strings.